docs/content/preview/contribute/core-database/build-from-src-almalinux.md

```diff
@@ -50,6 +50,53 @@ AlmaLinux 8 is the recommended Linux development platform for YugabyteDB.
 
 {{< /note >}}
 
+# TLDR
```

Comment:
In page headings start at level 2, that's what the styles support - bump all the headings down a level, or I provided an alternative